content/guides/models/registry/registry_cards.md

```diff
@@ -0,0 +1,78 @@
+---
+menu:
+  default:
+    identifier: registry_cards
+    parent: registry
+title: Document collections
```

Comment:
```suggestion
title: Annotate collections
```
Maybe this isn't quite the right edit, but there is some noun/verb ambiguity with "Document" here.